src/wp-admin/includes/media.php

```diff
@@ -2276,7 +2276,8 @@ function media_upload_form( $errors = null ) {
 	 * @since 2.6.0
 	 */
 	do_action( 'pre-html-upload-ui' ); // phpcs:ignore WordPress.NamingConventions.ValidHookName.UseUnderscores
-
+	$current_screen = get_current_screen();
+	$required       = $current_screen && 'media' === $current_screen->id ? ' required' : '';
```

Comment:
Thanks for checking `$current_screen` to make sure it is not `null`.

I think that the blank line after `do_action` is worth keeping, and the space could be after 'required':
```suggestion

	$current_screen = get_current_screen();
	$required       = $current_screen && 'media' === $current_screen->id ? 'required ' : '';
```

Reply:
right! done.